README.md

```diff
@@ -19,7 +17,7 @@ python text2image.py --prompt="An astronaut riding a horse"
 3) Using the python interface:
 
 ```
-pip install git+https://github.com/fchollet/stable-diffusion-tensorflow
+pip install git+https://github.com/divamgupta/stable-diffusion-tensorflow
 ```
```

Comment:
@PabloRosales 
This `pip install` can be separated from **Usage** option with its own section, i.e **Installation**. 

```
# Introduction
...
# Colab
...
# Installation 
...
# Usage
...
# Acknowledgement
...
```